terraform.tfvars.example

```diff
@@ -40,4 +42,5 @@ oci_base_bastion = {
   ssh_private_key_path      = ""
   ssh_public_key_path       = ""
   timezone                  = "Australia/Sydney"
+  use_autonomous            = false  
```

Comment:
Instead of adding the `use_autonomous` variable, could you write IF logic to use the Autonomous Linux image if the user leaves `bastion_image_id` at the default value of `NONE`?

Also, what's the behavior if the user specifies a bastion image AND sets `use_autonomous` to `true`? or if the user leaves variables both at their default values -- i.e., no image specified?

Reply:
This is a temporary measure. Either use Autonomous, Oracle Linux platform image or a custom image id. Once Autonomous is available as a platform image, it will be an either/or between Autonomous and custom image_id

The 2 settings are mutually exclusive (documented). However, the check is done on the use_autonomous value.